### ISSUE TITLE
python not a deb package in Ubuntu 22.0

### ISSUE THREAD

**Issue by kinostl** [piku/piku-bootstrap#4]
Ubuntu 22 only comes with python3 and now has a symlink package python-is-python3 and requires explicitely using python2 or python3. 

This can be fixed by changing the piku.yml file to something like this. It might also be possible to just remove the python dependency? I'm not sure about the internals but everything looks python3 so I assume the symlink is safe enough.

```yml
    - name: Install Debian Packages
      apt:
        pkg: ['bc', 'git', 'build-essential', 'libpcre3-dev', 'zlib1g-dev', 'python-is-python3', 'python3', 'python3-pip', 'python3-click', 'python3-dev', 'python3-virtualenv', 'python3-setuptools', 'nginx', 'acl', 'uwsgi-core', 'uwsgi-plugin-python3', 'nodeenv']
        update_cache: true
        state: present
```

**Comment by chr15m** [comment]
Thanks for flagging this. I'll have to work out how to accommodate older Ubuntu at the same time.

**Comment by btaylor84** [comment]
After seeing the comments in HN, I am spinning up a VM with Ubuntu 22.04 to try piku, and I ran into this error as well.

I'm proceeding with the edit mentioned above.

**Comment by chr15m** [comment]
Note to self: I think we can use `{{ ansible_distribution_major_version }}` to conditionally install the `python3` packages on Ubuntu 22 and above.

**Comment by chr15m** [comment]
I started setting up a GitHub action to test different Ubuntu versions on the branch called `automated-testing`. Once that's done I'll be able to put a fix in place that works on Ubuntu 18, 20, 22.

**Comment by chr15m** [comment]
This is fixed now, thanks all! Tested on Ubuntu 22.04 and it works well. There is an issue with the GitHub action test runners but I will look into that later.

**Comment by chr15m** [comment]
Fixed in dab3402e628979c6ba0752298e5ba1ab8571ff86.
